src/multiplayer-servers/getting-started/using-the-agones-sdk.md

```diff
@@ -80,3 +80,22 @@ Shutdown() instead. This will cause the hosting environment to properly clean up
 game server and send a termination signal to the process, which can then be used to
 shut it down.
 
+## Player count and capacity tracking
+
+Game Servers created by GameFabric have a counter for players by default which you can use to track player numbers and capacity per game server.
+
+
+### How to update
+Please see the [official Agones SDK documentation](https://agones.dev/site/docs/guides/client-sdks/).
+
+Name of the counter: `players`
+
+Note that a Counter Capacity of 0 means no limits. When incrementing a counter beyond its capacity, Agones will return an error.
+
+The Agones Counters and Lists API is currently in Beta state.
+
+curl example:
+`curl -d '{"count": "17", "capacity": "64"}' -H "Content-Type: application/json" -X PATCH http://localhost:${AGONES_SDK_HTTP_PORT}/v1beta1/counters/players`
```

Comment:
I assume using `localhost` works in the game server container or a sidecar container to send the request to the `agones-gameserver-sidecar` sidecar container, but I'm not 100% certain, so I left out the part that specified where exactly this command can be executed.
And using triple backquotes formats the code as a block instead of inline usually enabling stuff like copy buttons. Adding `bash` also adds bash syntax highlighting (if the frontend supports that)
```suggestion
Example request to set the player count and capacity using the standard linux command line tool `curl`:

```bash
curl -d '{"count": "17", "capacity": "64"}' -H "Content-Type: application/json" -X PATCH http://localhost:${AGONES_SDK_HTTP_PORT}/v1beta1/counters/players
```
```

Reply:
In case GitHub swallows the ending triple backquotes because it's identical to GitHub's suggestion syntax, the whole block should start and end with triple backquotes ``` and look like this:

```bash
curl -d '{"count": "17", "capacity": "64"}' -H "Content-Type: application/json" -X PATCH http://localhost:${AGONES_SDK_HTTP_PORT}/v1beta1/counters/players
```